doc/api/dgram.md

```diff
@@ -185,6 +185,27 @@ server.bind(41234);
 // Prints: server listening 0.0.0.0:41234
 ```
 
+#### Change to asynchronous `socket.bind()` behavior
+
+As of Node.js v0.10, [`dgram.Socket#bind()`][] changed to an asynchronous
+execution model. Legacy code would use synchronous behavior:
```

Comment:
I feel like Node v0.10 might be long ago by now that this can just be a YAML `changes:` entry (without examples)

Reply:
Is there a simple way to find a PR URL for YAML with this old change?

Reply:
Also, is this change also relevant for the next `socket.bind(options[, callback])` signature? it was added in v0.11.14, i.e. in the version where the previous signature was already asynchronous, so may by it is not relevant. But I may miss simething.

Reply:
I’d omit it, because that was before the time that Node.js necessarily had PRs for commits :)

332fea5ac1816e498030109c4211bca24a7fa667 was the change itself and 71694361f97035555a269625f68cfbd06effe58a added the docs for that (so, like, ~7 years ago).

Reply:
Ammended to make it a YAML section for the first (older) signature.